Detector/DetCommon/DetCommon/DetUtils.h

```diff
@@ -0,0 +1,12 @@
+#ifndef DETECTOR_DETUTILS_H
+#define DETECTOR_DETUTILS_H
+
+#include "DD4hep/DetFactoryHelper.h"
+
+namespace det {
+DD4hep::XML::Component getNodeByStrAttr(const DD4hep::XML::Handle_t& mother, const std::string& nodeName, const std::string& attrName,
+                                        const std::string& attrValue);
+
+double getDoubleWithDefault(const DD4hep::XML::Component& node, const std::string& attrName, const double& defaultValue);
```

Comment:
I will move the comment from the body here! I don't know a better name though - the xml method is called getAttr and I call it several times and use a default when it is not available.